nsls2forge_utils/dashboard.py

```diff
@@ -5,6 +5,35 @@
 functions had to be re-implemented here.
 """
 from .all_feedstocks import get_all_feedstocks
+from .meta_utils import get_attribute
+
+
+def _extract_org(pkg):
+    try_dev_url = False
+    # get org from home url
+    home_str = get_attribute('about home', pkg, 'nsls-ii-forge')
+    if home_str is None:
+        try_dev_url = True
+    elif 'github.com' in home_str:
+        split_str = home_str.split('/')
```

Comment:
It would not work as expected if the home URL contains the trailing slash.
Demo:
```py
2020-08-03T16:11:13.126581 [8] ▶ 'https://github.com/org/repo'.split('/')[-2]
Out[8]: 'org'  # correct

2020-08-03T16:11:16.588470 [9] ▶ 'https://github.com/org/repo/'.split('/')[-2]
Out[9]: 'repo'  # wrong
```

Solution:
```py
2020-08-03T16:11:31.346423 [10] ▶ 'https://github.com/org/repo'.rstrip('/').split('/')[-2]
Out[10]: 'org'  # correct

2020-08-03T16:11:35.912881 [11] ▶ 'https://github.com/org/repo/'.rstrip('/').split('/')[-2]
Out[11]: 'org'  # correct
```
It's cleaner and makes the code on lines 20-21 unnecessary.

Reply:
The code on lines 20-21 were actually to cover the case when the `home:` or `dev_url:` only has the organization.

For example, the `03-id-hxn-collection-feedstock` has this `https://github.com/NSLS-II-HXN` as its home url.

I do agree with the changes and will make them, but should I keep the second check for cases such as the one described above?

Reply:
Changes have been made [here](https://github.com/nsls-ii-forge/nsls2forge-utils/pull/34/commits/6ff7b0df2e1f7882f1ed52a5ab58ff23f5736c25) but I left in the second check for the case i described above.

Reply:
Yes, makes sense.